sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLListenerSuite.scala

```diff
@@ -343,6 +343,8 @@ class SQLListenerMemoryLeakSuite extends SparkFunSuite {
       .set("spark.sql.ui.retainedExecutions", "50") // Set it to 50 to run this test quickly
     val sc = new SparkContext(conf)
     try {
+      // Clear the sql listener created by a previous test suite.
+      SQLContext.clearSqlListener()
```

Comment:
Previously each `SQLContext` has its own `sqlListener`. Because now the SQL events are posted to the event bus. All `SQLContext` now share a single `sqlListener`. I don't think a user need clear `SQLContext.sqlListener`. This is only used by the unit tests.